MFE/dashboard/src/view/widget/wizard/wizard.jsx

```diff
@@ -52,7 +74,35 @@ const Wizard = ({ initialValues, ...props }) => {
   const handleFormSubmit = value => {
     const isLastPage = page === React.Children.count(children) - 1;
     if (isLastPage) {
-      return onSubmit(value);
+      console.log(value);
+
+      if (value.general.name.length < 5) {
+        dispatchEvent(EVENT.GLOBAL_TOAST, {
```

Comment:
Validation has to be done by the widget and not within the wizard. Each widget has to define its rules.